### PR TITLE
Ean/spinner issues in ci

### PR DESCRIPTION
**Pull request by ean**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: client/go/cmd/login.go [hunk 1]
```diff
@@ -27,7 +37,57 @@ func newLoginCmd(cli *CLI) *cobra.Command {
...
+				return fmt.Errorf("could not start the authentication process: %w", err)
+			}
+
+			fmt.Printf("Your Device Confirmation code is: %s\n\n", state.UserCode)
+
+			fmt.Println("If you prefer, you can open the URL directly for verification")
+			fmt.Printf("Your Verification URL: %s\n\n", state.VerificationURI)
+
+			fmt.Println("Press Enter to open the browser to log in or ^C to quit...")
+			fmt.Scanln()
```
Comment (mpolden) [inline client/go/cmd/login.go:51]: These should print to `cli.Stdout` or just use `log.PrintXXX`. Same below.

File: client/go/cmd/root.go [hunk 2]
```diff
@@ -203,6 +205,19 @@ func (c *CLI) configureFlags() {
 	c.flags = &flags
 }
 
+func (c *CLI) configureSpinner() {
+	// Explicitly disable spinner for Screwdriver. It emulates a tty but
+	// \r result in a newline, and output gets truncated.
+	_, screwdrvier := os.LookupEnv("SCREWDRIVER")
```
Comment (mpolden) [inline client/go/cmd/root.go:211]: This should access the environment through `c.Environment`.

Comment (mpolden) [reply client/go/cmd/root.go:211]: Typo: s/screwdrvier/screwdriver/

File: client/go/cmd/login.go (hunk 1 above)
Comment (ean) [reply client/go/cmd/login.go]: Fixed, but we are left with the ugly fmt.Scanln()` duckling. But I leave that to be handled as part of the ongoing discussion we have about the UX of this command.

Comment (mpolden) [reply client/go/cmd/login.go]: `fmt.Fscanln(cli.Stdin)` should work.

**Review by mpolden — APPROVED** [review]

**Pull request merged**